src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -571,7 +573,7 @@ AthensSurfaceExamples class >> exampleDrawForm2 [
 		can paintTransform scaleBy: 3; rotateByDegrees: 30.
 		can draw ].
 	
-	Display getCanvas drawImage: surf asForm  at: 0@0
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?